kvbc/test/s3_storage_factory_test.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 VMware, all rights reserved
+/**
+ * Tests for S3StorageFactory
```

Comment:
Nit: Current tests are not related to the factory itself - instead, focus is on key generation. Maybe we can change the test name or include it in the existing `ecs_s3_test`. Just a nit...